### DISCUSSION TITLE
web-ext: location of _locales folder

### DISCUSSION THREAD

**Discussion by avalanche1** [parcel-bundler/parcel#5877] [Q&A] [answered]
I have tf folder structure:
```
    .
    ├── dist
      └── manifest.json
      └── ...
    ├── src
      └── _locales
        └── en
          └── messages.json
      └── manifest.json
      └── ...
```
Naturally, I have set up i18n keys in `manifest.json`:
```json
{
  "name": "__MSG_extnName__",
  "version": "1.2",
  "description": "__MSG_appDescription__",
  "default_locale": "en",
}
```
But parcel compiler throws this:
![image](https://user-images.githubusercontent.com/10534563/108354438-7f833500-71fa-11eb-851c-b6156855a492.png)

So, the question is: where should I put the `_locales` folder? Surely, not into `dist`?

For ref: https://developer.chrome.com/docs/webstore/i18n/#detailed-description

**Comment by mischnic** [comment]
Looking at the source, `_locales` should be in the same folder as manifest.json. Not sure why you're getting the error

**Comment by avalanche1** [comment, marked as answer]
Ok, on detailed inspection - it turned out: I had two `_` characters in `__locales`.
🤦🏽‍♂️
